templates/todo/api/common/openapi.yaml

```diff
@@ -79,13 +79,15 @@ components:
       description: The max number of items to returns in a result
       schema:
         type: number
+        default: 20
```

Comment:
FYI, this is a shared OpenAPI specification across all the different language templates that `azd` produces. Most other languages do not use OpenAPI generators and will not respect this change automatically.

Separately, I don't think the front-end react app will handle paging correctly without an additional change. @wbreza can correct me if I'm wrong.

We probably want to pull this out a separate change outside a refactor since this would a breaking change that requires coordination across languages + front-end web app.

Reply:
Looking at it again... Looks like this was already the existing Java behavior. I do think that long-term we want to paginate, and defining a number makes sense. We can make this change and chalk it up to both other API implementations + web app to support this in the future.